lib/rust/ensogl/core/src/system/gpu/data/texture/storage/gpu_only.rs

```diff
@@ -1,126 +0,0 @@
-//! This module defines a texture storage which do not keep local data. It only keeps a reference
-//! to GPU texture of a given size.
```

Comment:
Why is this file removed? It was unused? Same for other files in `data::texture`

Reply:
Storage types except `GpuOnly` were unused. If we ever need it, a controller that keeps a uniform up to date could provide the functionality while keeping `Texture` simple.